config/jobs/cert-manager/release/cert-manager-release-presubmits.yaml

```
@@ -20,7 +20,7 @@ presubmits:
         - ./test/presubmit.sh
         resources:
           requests:
-            cpu: 2
+            cpu: 3500m
```

suggestion: would be nice to add a note that explains why we picked `3500m` e.g.

```suggestion
            # 3500m was chosen because that allows us to fit two jobs onto one
            # n1-standard-8 node, taking into account the amount of CPU allocated
            # to the kubelet.
            cpu: 3500m
```

---

Added in #511